ci-operator/config/openshift-online/ocm-sdk-go/openshift-online-ocm-sdk-go-main.yaml

```diff
@@ -0,0 +1,41 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.23
+images:
+# found via https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/p/ and https://docs.fedoraproject.org/en-US/epel/available-packages/
+- dockerfile_literal: |
+    FROM src
+    RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+    RUN dnf install -y pass || true
+    RUN dnf install -y qrencode || true
+    RUN dnf install -y https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/q/qrencode-4.1.1-1.el9.x86_64.rpm https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/q/qrencode-libs-4.1.1-1.el9.x86_64.rpm https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/p/pass-1.7.4-7.el9.noarch.rpm || true
```

Comment:
Based on the log, this works and installing normally does not.  Any advice for how to install this better?  https://docs.fedoraproject.org/en-US/epel/getting-started/